contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -407,6 +416,7 @@ Here are a few exceptions to the rules above---some cases are temporary workarou
   then we should fallback to using object kind with suffix based on value type. For example `podName`, `podUID`.
 * When providing multiple indistinguishable values (for example list of evicted pods), then we can use plural version of
   argument name. For example we should use `pods` and not `podList`.
+* we can use `Klog.KObjSlice` for Kubernetes objects.
```

Comment:
IMHO, this is required here. This section talks about `Here are a few exceptions to the rules above`.